.goreleaser.yml

```diff
@@ -0,0 +1,52 @@
+project_name: driftctl
+dist: bin
+release:
+    github:
+        owner: cloudskiff
+        name: driftctl
+    draft: false
+    prerelease: auto
+before:
+    hooks:
+        - go mod tidy
```

Comment:
I don't think that it is the build to handle this. The go mod tidy should be something done by the developpers.